packages/sentry/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as Sentry from '@sentry/nextjs';
+
+export const sentryFetch: typeof fetch = async (resource, options) => {
+   const context = {
+      // Underscore sorts the resource first in Sentry's UI
+      _resource: resource,
+      headers: options?.headers,
+      method: options?.method,
+      // Parse to pretty print GraphQL queries
+      body: options?.body ? JSON.parse(String(options?.body)) : undefined,
+   };
+   return fetch(resource, options)
+      .then((response) => {
+         if (response.status >= 400 && response.status !== 401) {
+            const msg = `\`fetch()\` HTTP error: ${response.status} ${response.statusText}`;
```

Comment:
```suggestion
            const msg = `fetch() HTTP error: ${response.status} ${response.statusText}`;
```

Did the backticks do something special in the error message in sentry? Otherwise, I think they are superfluous.

Reply:
Nope they don't get rendered as anything special. Happy to drop them